### DISCUSSION TITLE
实现类似Ant Design中水印的效果

### DISCUSSION THREAD

**Discussion by mmzhw** [zhensherlock/watermark-js-plus#12] [Q&A] [answered]
<img width="1170" alt="image" src="https://github.com/zhensherlock/watermark-js-plus/assets/31273427/7813b395-5a1b-4598-a6e0-608a6d86d33b">

**Comment by zhensherlock** [comment, marked as answer]
const watermark = new Watermark({
  layout: 'grid',
  gridLayoutOptions: {
    rows: 2,
    cols: 2,
    matrix: [[1,0], [0,1]]
  }
})
添加上述参数，可实现该效果

**Comment by mmzhw** [comment]
感谢
